README.md

```diff
@@ -179,35 +180,36 @@ Always assigned to a host and is ghosted from the start.<br>
 Team : Impostors<br>
 Basis : Impostor<br>
 
-If the BountyHunters kill their designated target, their next kill cooldown will be much less than usual.<br>
-Killing a player except their current target results in an increased kill cooldown.<br>
+If the BountyHunter kill their designated target, their next kill cooldown will be much less than usual.<br>
```

Comment:
To make this grammatically correct in English, you want it to read like this:
Singular BountyHunter: "If the BountyHunter kill's its designated target..."
Plural BountyHunter: "If the BountyHunters' kill their designated target..."